docs/src/howto.md

```diff
@@ -121,3 +121,96 @@ Action(1:100, (args...)->rect(pos(:my_circle)+Point(-10, -100), 20, 20, :fill))
 
 In this animation the position of the circle is saved inside `:my_circle` and can be used with `pos(:my_circle)` inside the `rect` function.
 
+## How can I show a text being drawn?
+
+A `text` can appear as every other object with `appear(:fade)` and `appear(:scale)` but it also has a special [`appear`](@ref) functionality called 
+`:draw_text`.
+
+You can use 
+```julia
+Action(
+    1:100,
+    (args...) -> text("Hello World!"; halign = :center);
+    subactions = [
+        SubAction(1:15, sineio(), appear(:draw_text)),
+        SubAction(76:100, sineio(), disappear(:draw_text)),
+    ]
+)
+```
+
+to let the text `"Hello World!"` appear from left to right in an animated way. 
+
+## How can I let an object move along a path?
+
+In this case we need to create a list of points (a path) that the object can follow.
+This can be simple object like a `star` or just a list of points. 
```

Comment:
"This can be simple object like a `star` or just a list of points. " -> "The path can be a provided object path such as `star` or list of points."

Perhaps link to these object paths too?

Reply:
What do you mean by linking to it?